wp-includes/widgets/class-wp-widget-visual-text.php

```diff
@@ -149,18 +154,31 @@ public function form( $instance ) {
 				'text' => '',
 			)
 		);
+		?>
+		<input id="<?php echo $this->get_field_id( 'title' ); ?>" name="<?php echo $this->get_field_name( 'title' ); ?>" class="title" type="hidden" value="<?php echo esc_attr( $instance['title'] ); ?>">
+		<input id="<?php echo $this->get_field_id( 'text' ); ?>" name="<?php echo $this->get_field_name( 'text' ); ?>" class="text" type="hidden" value="<?php echo esc_attr( $instance['text'] ); ?>">
```

Comment:
There is a chance that existing plugins that try to extend the Text widget could trip over this here. If they try to obtain a textarea in here, they will fail. But then again, the textarea's behavior is radically changed with the introduction of TinyMCE, so perhaps it's a good thing that it is moved out of the way to isolate it.